### ISSUE TITLE
Version number in custom building

### ISSUE THREAD

**Issue by byzheng** [APSIMInitiative/ApsimX#9542]
### Describe the new feature

`Models --version` will display the version number. However it will show APSIM 0.0.0.0 for custom building. 

Is it possible to display the actual version number or build number? The reason I am asking is as it will be useful to develop third party application (e.g. R) to check support version.

**Comment by par456** [comment]
Not really, the code in the git repo doesn't track what the last PR merged in was and what build number it was assigned, that is all done in the build scripts when the release builds are created.

If someone is working in a debug version of apsim, you really can't assume anything about the code they are working with, even if you knew when their branch was last updated from main. 

What's the reasoning behind trying to track that version number? Is that for checking other user's, or your own simulations? We might be able to suggest a better alternative to look for.

**Comment by byzheng** [comment]
In my R package, I provided codes to generate new apsimx file (i.e. cultivar parameters and new models). But this function depends on the actual of apsimx file. I would like to check the version number of Models to decide whether the following codes should be processed. 

I understand the file number in the apsimx indicate the file structure, but which is also not always available especially part of apsimx file.
